mithril-aggregator/src/database/provider/cardano_transaction.rs

```diff
@@ -231,28 +246,32 @@ impl CardanoTransactionRepository {
     }
 
     /// Return the [CardanoTransactionRecord] for the given transaction hash.
-    pub async fn get_transaction(
+    pub async fn get_transaction<T: Into<TransactionHash>>(
```

Comment:
There are two calls of `get_transaction` that could be modified. If it's the case, same for the repository of the signer node.
- line 476 with `get_transaction("tx-hash-123")`
- line 649 with `get_transaction("tx-hash-000")`

Reply:
Done here and in signer node